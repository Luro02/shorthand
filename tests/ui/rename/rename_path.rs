use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename)]
    first: usize,
}

fn main() {}
