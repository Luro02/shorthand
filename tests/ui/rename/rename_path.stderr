error: unexpected meta-item format `Path`, expected `List`
 --> $DIR/rename_path.rs:5:17
  |
5 |     #[shorthand(rename)]
  |                 ^^^^^^
